src/runners/http_rfqm_service_runner.ts

```diff
@@ -113,12 +118,13 @@ export async function runHttpRfqmServiceAsync(
     rfqmService: RfqmService,
     configManager: ConfigManager,
     config: HttpServiceConfig,
+    connection: Connection,
     _app?: core.Express,
 ): Promise<{ app: express.Application; server: Server }> {
     const app = _app || express();
     app.use(addressNormalizer);
     const server = createDefaultServer(config, app, logger, async () => {
-        /* TODO - clean up DB connection when present */
+        await connection.close();
```

Comment:
Just to confirm, the `connection.close()` is run once the server terminates? is that right?

Reply:
correct!